src/Agent.Plugins/PipelineArtifact/PipelineArtifactPluginV1.cs

```diff
@@ -56,34 +58,22 @@ public class PublishPipelineArtifactTaskV1 : PipelineArtifactTaskPluginBaseV1
         protected override string TargetPath => "path";
 
         private static readonly Regex jobIdentifierRgx = new Regex("[^a-zA-Z0-9 - .]", RegexOptions.Compiled | RegexOptions.CultureInvariant);
+        private static readonly string pipelineType = "pipeline";
+        private static readonly string fileShareType = "filepath";
 
         protected override async Task ProcessCommandInternalAsync(
             AgentTaskPluginExecutionContext context, 
             CancellationToken token)
         {
             string artifactName = context.GetInput(ArtifactEventProperties.ArtifactName, required: false);
             string targetPath = context.GetInput(TargetPath, required: true);
+            string artifactType = context.GetInput(ArtifactEventProperties.ArtifactType, required: true);
```

Comment:
>required: true); [](start = 89, length = 16)

required: true might break because the task and agent deployments are not synced so you need to be able to handle mutliple cases. what's the plan here?

Reply:
That's true. We need to set it to false and default to pipeline for now I think.